new-lamassu-admin/src/routing/utils.js

```diff
@@ -0,0 +1,8 @@
+export const isLoggedIn = userData => {
+  return userData
```

Comment:
We either cast this into a boolean or just stop using this function. It's a identity function right now.

Reply:
You're correct. Changed to 
```suggestion
  return !!userData
```